src/sentry_options.h

```diff
@@ -14,6 +14,26 @@ class SentryOptions {
 	using GodotErrorType = sentry::GodotErrorType;
 	using GodotErrorMask = sentry::GodotErrorMask;
 
+public:
+	struct LoggerLimits {
+		// TODO: Establish proper default limits with the Sentry team.
+
+		// Limit the number of lines that can be parsed per frame.
+		int parse_lines = 30;
+
+		// Protect frametime budget.
+		int breadcrumbs_per_frame = 5;
+		int events_per_frame = 1;
+
+		// Limit to 1 error captured per source line within T milliseconds window.
+		int repeated_error_window_ms = 1000;
+
+		// Limit to N events and M breadcrumbs within T milliseconds window.
+		int throttle_events = 4;
+		int throttle_breadcrumbs = 20;
+		int throttle_window_ms = 10000;
+	};
```

Comment:
@bruno-garcia @bitsandfoxes 
Looking for feedback on the defaults for these limits.

Reply:
We opted to default to 1/second in the Unity SDK. But that's a fairly magic number.